modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryListeningTimeView.kt

```diff
@@ -32,22 +60,119 @@ fun StoryListeningTimeView(
             context.resources
         )
     }
-    Column(modifier.padding(16.dp)) {
-        TextH30(
-            text = stringResource(LR.string.end_of_year_listening_time, timeText),
-            textAlign = TextAlign.Center,
-            color = story.tintColor,
-            modifier = modifier
-                .fillMaxWidth()
-                .padding(16.dp)
-        )
-        TextH30(
-            text = funnyText,
-            textAlign = TextAlign.Center,
-            color = story.tintColor,
-            modifier = modifier
-                .fillMaxWidth()
-                .padding(16.dp)
-        )
+
+    val backgroundColor = if (story.podcasts.isEmpty()) {
+        Podcast().getTintColor(false)
+    } else {
+        // Get the middle podcast
+        story.podcasts.take(2).last().toPodcast()
+            .getTintColor(false)
     }
+    Column(
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = Color(backgroundColor))
+            .verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = modifier.weight(1f))
+
+        Title(timeText, story, modifier)
+
+        FunnyText(funnyText, story, modifier)
+
+        Spacer(modifier = modifier.weight(1f))
+
+        PodcastCoverRow(story, modifier)
+
+        Spacer(modifier = modifier.weight(1f))
+
+        Logo(modifier)
+    }
+}
+
+@Composable
+private fun Title(
+    timeText: String,
+    story: StoryListeningTime,
+    modifier: Modifier,
+) {
+    TextH20(
+        text = stringResource(LR.string.end_of_year_listening_time, timeText),
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(start = 40.dp, end = 40.dp)
+    )
+}
+
+@Composable
+private fun FunnyText(
+    funnyText: String,
+    story: StoryListeningTime,
+    modifier: Modifier,
+) {
+    TextP40(
+        text = funnyText,
+        textAlign = TextAlign.Center,
+        color = story.tintColor,
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(top = 24.dp, start = 40.dp, end = 40.dp)
+    )
```

Comment:
Can we give this 80% opacity to match the design, also should we make it slightly bolder.

<img width="687" alt="Screenshot 2022-11-10 at 2 41 08 pm" src="https://user-images.githubusercontent.com/308331/200998819-152c5e52-de61-444d-b6f9-c38c9684ec91.png">

<img width="636" alt="Screenshot 2022-11-10 at 2 41 17 pm" src="https://user-images.githubusercontent.com/308331/200998840-7f529627-d5b2-4cc7-bb96-1e3ee2fb6c27.png">

Reply:
Nice observation! Updated in [1137927](https://github.com/Automattic/pocket-casts-android/pull/556/commits/11379277f67ca899cab820bf5acb43fefcda599d).